modules/host/kernel/virtualization.config

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: CC-BY-SA-4.0
+CONFIG_IKCONFIG=y
+CONFIG_IKCONFIG_PROC=y
+CONFIG_POSIX_TIMERS=y
+CONFIG_PRINTK=y
+CONFIG_BASE_FULL=y
+CONFIG_VIRTUALIZATION=y
+CONFIG_HIGH_RES_TIMERS=y
+CONFIG_KVM=y
+CONFIG_KVM_INTEL=y
+CONFIG_KSM=y
+CONFIG_INPUT_EVDEV=y
+CONFIG_I2C=y
+CONFIG_I2C_DESIGNWARE_PLATFORM=y
+CONFIG_I2C_HID_ACPI=y
```

Comment:
What's the case with `I2C*` specifics? General or Lenovo X1 related?

Reply:
Ok, as I suspected. These are `gui-vm`-specific and input-devices are not needed in other guests. They can also get very device (both input device and target device) specific depending on the target configuration.  

All-in-all, they are not needed in all the guest VMs. Can you please refactor these to `user-input-devices.config` and include the config fragment only in the GUI VM?

Reply:
Done refactoring as per review comment.